<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QToolButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
    <property name="answer" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QToolButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
    <property name="answer" stdset="0">
     <number>2</number>
    </property>
   </widget>
   <widget class="QToolButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
    <property name="answer" stdset="0">
     <number>3</number>
    </property>
   </widget>
   <widget class="QToolButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
    <property name="answer" stdset="0">
     <number>4</number>
    </property>
   </widget>
   <widget class="QStackedWidget" name="qstWdg">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>501</width>
      <height>231</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="text">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="txtQst">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>451</width>
        <height>131</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="picture">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="imgLbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>451</width>
        <height>181</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="imgQst">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>20</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sound">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <widget class="QToolButton" name="playBtn">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="stopBtn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="musicQst">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="score">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="endLbl">
      <property name="geometry">
       <rect>
        <x>73</x>
        <y>110</y>
        <width>311</width>
        <height>81</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="timerLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QLabel" name="mouseCoord">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>389</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
